src/core/udb-api.service.js

```diff
@@ -163,8 +163,8 @@ function UdbApi(
       searchParams.q = queryString;
     }
 
-    if (showDrafts) {
-      searchParams.workflowStatus = 'DRAFT,' + searchParams.workflowStatus;
+    if (showUnavailable) {
+      searchParams.workflowStatus = 'DRAFT,REJECTED,DELETED,' + searchParams.workflowStatus;
```

Comment:
```suggestion
      searchParams.workflowStatus = [
          ...(new Set('DRAFT', 'REJECTED', 'DELETED', searchParams.workflowStatus));
      ].join(',');
```

Maybe we should avoid duplicates. If we can't use Set we could use lodash unique or something

Reply:
Oh yeah good point